update ingress snippet comment in values.yaml

Northcode opened the pull request:
# Pull Request

## Description of the change

Update well-known webfinger and nodeinfo redirects to match documentation in https://docs.nextcloud.com/server/23/admin_manual/issues/general_troubleshooting.html#service-discovery

## Benefits

People just uncommenting the snippet will now have correct webfinger config, and no warnings in the nextcloud settings.

## Possible drawbacks

None that I can see

## Applicable issues

#240 

## Additional information

...

## Checklist

- [X] DCO has been [signed off on the commit](https://docs.github.com/en/github/authenticating-to-github/signing-commits).
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).

Jeroen0494 commented:
@Northcode any change you could revisit this PR? I can confirm this fix works.

Northcode commented:
@Jeroen0494 
Hi. Sorry for the late reply. I had forgotten about this.
I've fixed the things jessebot was complaining about and rebased on upstream.
Not sure how to proceed as github still says jessebot is requesting changes.

Northcode commented:
@tvories 
I think it was complaining that there was trailing whitespace on line 35 of the [old commit](https://github.com/nextcloud/helm/commit/c3aaa3d8b7bffdc6259060c498e71c36c24782b4).
I've corrected that in my [updated code](https://github.com/nextcloud/helm/commit/7e209a1a453bd4faee4a6d33f9baf1d7d3dc3330), but it doesn't look like jessebot has rerun, its still complaining about the old commit.
I marked those corrections are resolved, is there something else I have to do for it to detect my changes?

Jeroen0494 commented:
> I marked those corrections are resolved, is there something else I have to do for it to detect my changes?

You can go to the "1 change requested" option, click the menu and select "Request re-review" or something.
![image](https://user-images.githubusercontent.com/9626301/228042779-cb0b95d4-7a1e-444f-b6dd-d572c47ebdda.png)

tvories commented on charts/nextcloud/values.yaml:
```diff
@@ -31,7 +31,9 @@ ingress:
-  #    rewrite ^/.well-known/webfinger /public.php?service=webfinger last;
+  #    rewrite ^/.well-known/webfinger /index.php/.well-known/webfinger last;
+  #    rewrite ^/.well-known/nodeinfo /index.php/.well-known/nodeinfo last;
+
```
Is there any reason we need this newline here?  It makes sense to group the rewrites together.

Northcode commented:
@jessebot oh I see. In that case sorry for assuming your uhm.. type of consciousness? I guess? haha

Anyways I've removed the whitespace now on both line 36 and 33 and rebased onto upstream again, incrementing the chart version to 3.5.8
